calculator/main.py

```diff
@@ -13,6 +15,15 @@ def create_new_calculator(operations=None):
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
     """
+    if operations == None:
```

Comment:
It'd be easier to define `operations` as an empty dictionary in the `create_new_calculator` header, unless there's a an unforeseen reason for doing it this way?